src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -82,11 +83,44 @@ export default function OpeningPeriodForm({
   errorTextAndLabel,
 }: OpeningPeriodFormProps): JSX.Element {
   const language = Language.FI;
-  const {
-    name: nameFieldConfig,
-    resourceState: resourceStateConfig,
-    timeSpanGroup: { rule: ruleConfig },
-  } = datePeriodConfig;
+
+  const nameFieldConfig = datePeriodConfig.name;
+
+  const resourceStateConfig: UiFieldConfig = {
+    options: datePeriodConfig.resourceState.options.map(
+      (translatedApiChoice) => ({
+        value: translatedApiChoice.value,
+        label: translatedApiChoice.label.fi as string,
```

Comment:
Is it possible to use the language constant in these?
```js
translatedApiChoice.label[language]
```

Reply:
- [ ] Apply language constant